dir.traversal.targets

```diff
@@ -141,31 +141,19 @@
           BeforeTargets="TestAllProjects"
           Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
 
-    <PropertyGroup>
-      <TestTFMSupportSet>$(FilterToTestTFM)</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='netcoreapp1.1'">$(TestTFMSupportSet)|netcoreapp1.0</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net461'">$(TestTFMSupportSet)|net46</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net462'">$(TestTFMSupportSet)|net461|net46</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net463'">$(TestTFMSupportSet)|net462|net461|net46</TestTFMSupportSet>
-    </PropertyGroup>
-
     <ItemGroup>
       <Project>
-        <!-- default to netcoreapp1.0.0 if TestTFMs aren't set on the project -->
-        <TestTFMs Condition="'%(Project.TestTFMs)'==''">netcoreapp1.0</TestTFMs>
+        <!-- default to netcoreapp1.1 if TestTFMs aren't set on the project -->
+        <TestTFMs Condition="'%(Project.TestTFMs)'==''">netcoreapp1.1</TestTFMs>
```

Comment:
Actually we might want to think about this one a little more (especially in cases where people actually set it at the commandline and it isn't the default) but I do still think it might make since to share a common property for the default. Perhaps we should create an independent "DefaultTestTFM" property and use that as the default in all these places.